src/lib/API/api.js

```diff
@@ -387,13 +387,6 @@ export class APIService {
     return this.client.get('/verify/w3/bonuses')
   }
 
-  /**
```

Comment:
could we also remove hanuka-related stuff from the GoodServer ?

Reply:
yes